src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.ts

```diff
@@ -14,77 +14,77 @@ import {
   ProceedingsOrderTypes,
   Respondent,
 } from '../../../app/case/definition';
-import { AppRequest } from '../../../app/controller/AppRequest';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-export const setProceedingDetails = (UserCase, respondent: Respondent, req: AppRequest): Respondent => {
-  const respondentDetails = respondent;
+export const setProceedingDetails = (userCase: CaseWithId): CurrentOrPreviousProceedings => {
```

Comment:
Change the method name to prepareProceedingDetailsRequest

Reply:
Done Vivek.